src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/controllers/draft/DraftAdjudicationController.kt

```diff
@@ -143,6 +143,7 @@ class DraftAdjudicationController(
     ),
   )
   @GetMapping("/my-reports")
+  @PreAuthorize("hasRole('VIEW_ADJUDICATIONS')")
```

Comment:
For each `GET` endpoint without a scope or existing role, I added `VIEW_ADJUDICATIONS`